### ISSUE TITLE
Virtual Monitors with GNOME's Mutter on Wayland (also headless)

### ISSUE THREAD

**Issue by Jmennius** [pavlobu/deskreen#162]
With GNOME 40 came support for Virtual Monitors (and headless, which is another valuable use-case). See [Mutter MR 1698](https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/1698).
This is Wayland-specific and requires PipeWire for screen-casting (which should already work?).

I am basically dropping this here to just let people know that this is a possibility now. It's not likely that I'll try it and provide more information in foreseeable future...

**Comment by elfzweik** [comment]
Yes, GNOME 40 support Virtual Monitors. I tried vkms on X11. It works and deskreen works well on the virtual monitor created by vkms. Since the system changed from Wayland to X11, I felt obvious stalling on mouse movement not only the virtual display but all the displays. When changing back to wayland, things getting much better. The bad news is the [Mutter MR 1698](https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/1698). you mentioned, does not work for me. Mutter feed back 
```
(mutter:12075): libmutter-WARNING **: 03:24:11.939: WL: error: XDG_RUNTIME_DIR not set in the environment


(mutter:12075): libmutter-ERROR **: 03:24:11.939: Failed to create socket
```
 But the good news is that I succeeded in using the following command to create a virtual display.
```
gsettings set org.gnome.desktop.remote-desktop.rdp screen-share-mode extend
```
After executing the command, nothing happened to the display. I didn't see the virtual display. But when I turned on the remote desktop function of Ubuntu 22.04 and connected the computer with RDP client built in Windows 10, the virtual display showed up.as the extended display. The response on the extended display are much faster. BTW, deskreen did not work at this time, it cannot find the virtual display, either.
